taskcluster/kinds/train-teacher/kind.yml

```diff
@@ -101,6 +104,7 @@ tasks:
                 - >-
                     pip3 install --upgrade pip setuptools &&
                     pip3 install -r $VCS_PATH/pipeline/train/requirements/train.txt &&
+                    pip3 install $VCS_PATH/tracking &&
```

Comment:
Let's also add it to all other steps that use `train-taskcluster.sh` (train-*, finetune-*)